internal/database/sqlcommon/contractevents_sql_test.go

```diff
@@ -41,12 +41,13 @@ func TestContractEventsE2EWithDB(t *testing.T) {
 		Name:         "Changed",
 		Outputs:      fftypes.JSONObject{"value": 1},
 		Info:         fftypes.JSONObject{"blockNumber": 1},
+		Timestamp:    fftypes.Now(),
 	}
 
 	s.callbacks.On("OrderedUUIDCollectionNSEvent", database.CollectionContractEvents, fftypes.ChangeEventTypeCreated, "ns", event.ID, int64(1)).Return()
 
 	err := s.InsertContractEvent(ctx, event)
-	assert.NotNil(t, event.Created)
+	assert.NotNil(t, event.Timestamp)
```

Comment:
Small thing, but there's probably no need for this check now. It was verifying that the Created time was added by the db logic (but it's now passed in, not added).